specs/driver_spec.rb

```diff
@@ -77,4 +77,99 @@
       driver.average_rating.must_equal 0
     end
   end
+
+  describe "total_revenue" do
+    it "must calculate the total revenue for all trips for a given driver" do
+      # Test 1: three trips
+      trips = [
+        RideShare::Trip.new({cost: 5, rating: 3, start_time: "2016-08-08T12:14:+300:00", end_time: "2016-08-08T12:15:00+00:00"}),
+        RideShare::Trip.new({cost: 7, rating: 3, start_time: "2016-08-08T12:14:+300:00", end_time: "2016-08-08T12:15:00+00:00"}),
+        RideShare::Trip.new({cost: 8, rating: 3, start_time: "2016-08-08T12:14:+300:00", end_time: "2016-08-08T12:15:00+00:00"})
+      ]
+
+      driver_data = {
+        id: 7,
+        vin: 'a' * 17,
+        name: 'test driver',
+        trips: trips
+      }
+
+      driver = RideShare::Driver.new(driver_data)
+      driver.total_revenue.must_equal 12.04
+    end
+  end
+
+  describe "finished_trips" do
+    it "return an array of trips that have end time" do
+      trips = [
+        RideShare::Trip.new({cost: 5, rating: 3, start_time: "2016-08-08T12:14:+300:00", end_time: "2016-08-08T12:15:00+00:00"}),
+        RideShare::Trip.new({cost: 7, rating: 3, start_time: "2016-08-08T12:14:+300:00", end_time: "2016-08-08T12:15:00+00:00"}),
+        RideShare::Trip.new({cost: 8, rating: 3, start_time: "2016-08-08T12:14:+300:00", end_time: nil})
+      ]
+      driver_data = {
+        id: 7,
+        vin: 'a' * 17,
+        name: 'test driver',
+        trips: trips
+      }
+
+      driver = RideShare::Driver.new(driver_data)
+      driver.finished_trips.length.must_equal 2
+      trips.last.finish_trip!
+      driver.finished_trips.length.must_equal 3
```

Comment:
Good work testing this helper method - I like that this covers both with and without incomplete trips. I would also be interested in what happens when there are no complete trips.